readme.md

```diff
@@ -10,30 +12,32 @@ Pairs very well with the [key-master](https://github.com/TehShrike/key-master).
 npm install gate-keeper
 ```
 
-`var gateKeeper = require('gate-keeper')`
+`const gateKeeper = require('gate-keeper')`
 
 # Example
 
 <!-- js
-var gateKeeper = require('./')
+const gateKeeper = require('./')
 -->
 
 ```js
 let calledAlready = false
-const get = gateKeeper(function getSomeRemoteResource(cb) {
+const get = gateKeeper(function getSomeRemoteResource(state) {
```

Comment:
This would read better for me with destructuring: `getSomeRemoteResource({ isCancelled })`. When I first read `state`, it surprised me until I remembered the function signature from the docs. Reading the usage a few lines down is also satisfactory, but I like to read a detailed function signature up front.

Reply:
Good point, no reason to give that map a name.